UIKIT_DZ_Clone_AppStore/UIKIT_DZ_Clone_AppStore/ViewControllers/ProductViewController/ProductViewController.swift

```diff
@@ -60,6 +60,19 @@ final class ProductViewController: UIViewController {
         sender.layer.borderWidth = 3
     }
 
+    @objc private func showWebView(sender: Any) {
```

Comment:
```suggestion
    @objc private func showWebView(sender: UIButton) {
```
не проще сразу так написать?